src/main/java/org/jitsi/videobridge/health/Health.java

```diff
@@ -112,19 +112,11 @@ private static void check(Conference conference)
 
         for (int i = 0; i < numEndpoints; ++i)
         {
-            Endpoint endpoint
-                = conference.getOrCreateLocalEndpoint(generateEndpointID());
-
-            // Fail as quickly as possible.
-            if (endpoint == null)
-            {
-                throw new NullPointerException("Failed to create an endpoint.");
-            }
-
-            // Trigger the creation of the transport manager.
+            final Endpoint endpoint;
             try
             {
-                endpoint.getTransportManager();
+                endpoint = conference.createLocalEndpoint(
+                    generateEndpointID(), true);
```

Comment:
I know we're not currently running ICE, but should we have different roles for the two endpoints?

Reply:
I don't know what is correct, but you've just described behavior of `JVB 1.0` :) - so I've just added the same behavior to `JVB 2.0`
https://github.com/jitsi/jitsi-videobridge/blob/838e190c4dcf0ca5b94a4065780a59a82792a744/src/main/java/org/jitsi/videobridge/health/Health.java#L141-L143